app/controllers/customers_controller.rb

```diff
@@ -61,14 +68,30 @@ def destroy
     end
   end
 
-  private
-    # Use callbacks to share common setup or constraints between actions.
-    def set_customer
-      @customer = Customer.find(params[:id])
-    end
+  def phone_exist(number)
+    Customer.find_by('phone_number = ?', number)
```

Comment:
@kalgibhatt-btc If you just want to check existence, should fire count based query only. - `.count > 0` or `.exists?`